### ISSUE TITLE
Parent directory button in catalog.html 

### ISSUE THREAD

**Issue by PBrockmann** [Unidata/tds#337]
In 2016, this question was asked:
"Could it be possible to have a parent dir button in every sub dir when using a datasetScan in thredds ?"
https://github.com/Unidata/thredds/issues/590

Is it possible today ?

**Comment by haileyajohnson** [comment]
It's still not possible today, unfortunately. It is feasible to implement, and would actually be great for us for providing support as well, so that's good motivation, but we have a lot of balls in the air right now and I don't see it to the top of the stack of features to implement very soon. I'll leave this issue open to revisit :)

**Comment by PBrockmann** [comment]
Yes, interested to try to implement this basic missing feature. 
I will investigate the customization of page. 
https://docs.unidata.ucar.edu/tds/current/userguide/customizing_tds_look_and_feel.html

So it's all about Thymeleaf Templates.
Any recommendation ?

**Comment by haileyajohnson** [comment]
It is possible to add you own html using thymeleaf snippets, including navigations and links, however I think some code changes would need to make made such that the link the the parent dir is always available from a lower level dir or dataset context. That's where it get's hairy.
